services/blockchain-indexer/shared/dataService/business/pos/stakers.js

```diff
@@ -72,19 +74,42 @@ const getStakers = async params => {
 		const { name, ...remParams } = params;
 		params = remParams;
 
-		const { address } = await getIndexedAccountInfo({ name, limit: 1 }, ['address']);
+		const { address } = await getIndexedAccountInfo({ name }, ['address']);
 		params.validatorAddress = address;
-		stakers.meta.validator.name = name;
+		stakersResponse.meta.validator.name = name;
 	}
 
 	// If validatorAddress is unavailable, return empty response
-	if (!params.validatorAddress) return stakers;
-
-	// TODO: Use count method directly once support for custom column-based count added https://github.com/LiskHQ/lisk-service/issues/1188
-	const [{ numStakers }] = await stakesTable.rawQuery(`SELECT COUNT(stakerAddress) as numStakers from ${stakesIndexSchema.tableName} WHERE validatorAddress='${params.validatorAddress}'`);
+	if (!params.validatorAddress) return stakersResponse;
+
+	// Search param is used to filter stakers by name. Prepare list of valid staker addresses
+	// for the given search (staker name) param
+	const allowedStakersAddressNameMap = {};
```

Comment:
Please consider renaming `allowedStakersAddressNameMap` to something that indicates that it relates to the `search` param or to simply `stakerAddressNameMap`. Can cause confusion on L127 🙂

Reply:
Applied.